exercises/practice/leap/.approaches/config.json

```
@@ -0,0 +1,32 @@
+{
+  "introduction": {
+    "authors": ["meatball133"],
+    "contributors": ["BNAndras"]
+  },
+  "approaches": [
+    {
+      "uuid": "5d42dc83-2473-425a-90bd-bf03f92b8c8b",
+      "slug": "divisible-by",
+      "title": "Use divisible-by",
+      "blurb": "Use divisible-by to check number.",
```

I don't think `divisible-by` is the defining characteristic of the approach (the other two approaches use it too), but instead it is the fact that a chain of boolean expressions is used:

```suggestion
      "slug": "boolean-chain",
      "title": "Use boolean chain",
      "blurb": "Use a chain of boolean expressions.",
```